sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -513,6 +518,14 @@ class RapidsExecutorPlugin extends ExecutorPlugin with Logging {
     extraExecutorPlugins.foreach(_.shutdown())
     FileCache.shutdown()
     GpuCoreDumpHandler.shutdown()
+
+    val nsysStopCommand = "nsys stop"
+    val result: String = nsysStopCommand.!!
+    println(s"Nsys Stop Command output: $result")
+    Thread.sleep(120* 1000)
```

Comment:
Will this make the GPU perf worse ?

Reply:
Not really. only enabling cuda memory track(`--cuda-memory-usage=true`) will make perf worse, but I didn't enable that. 
Should we enable that? that is used to see peak memory stuff.

Reply:
i mean the "sleep" here, it will take 2 mins.

Reply:
Oh I got your point. I removed the sleep, as the command itself is blocking, no need to wait.
And as we discussed offline that the report generation time is counted, so for analysis, use the new branch, for perf bench, use the orignal one which doesn't do nsys.